src/main/kotlin/uk/gov/dluhc/notificationsapi/messaging/mapper/TemplatePersonalisationMessageMapper.kt

```diff
@@ -241,9 +282,9 @@ abstract class TemplatePersonalisationMessageMapper {
 
     protected fun mapSignatureRejectionReasons(
         languageDto: LanguageDto,
-        personalisation: RejectedSignaturePersonalisation,
+        rejectionReasons: List<SignatureRejectionReason>,
     ): List<String> {
-        return personalisation.rejectionReasonsExcludingOther.map { reason ->
+        return rejectionReasons.filter { it !== SignatureRejectionReason.OTHER }.map { reason ->
```

Comment:
Very typescript of you
```suggestion
        return rejectionReasons.filter { it != SignatureRejectionReason.OTHER }.map { reason ->
```